src/views/Give/RecipientModal.tsx

```diff
@@ -92,7 +92,7 @@ export function RecipientModal({
    * TODO consider extracting this into a helper file
    */
   const sohmBalance: string = useSelector((state: State) => {
-    return state.account.balances && state.account.balances.mockSohm;
+    return state.account.balances && networkId === 1 ? state.account.balances.sohm : state.account.balances.mockSohm;
```

Comment:
Is mockSohm guaranteed to be valid if the networkId is NOT 1? I recall that you added a case for value 1 and 4, but others were skipped. So consider flipping this around. If networkId === 4, use mockSohm (since that's the exception), otherwise act as normal.